dlrs/main/classes/RollupService.cls

```diff
@@ -1548,6 +1548,8 @@ global with sharing class RollupService {
     // Unable to lower RelationShipCriteria__c because of field value case-(in)sensitivity configuration
     return (lookupWrapper.Lookup.ParentObject.toLowerCase() +
     '#' +
+    lookupWrapper.Lookup.ChildObject.toLowerCase() +
```

Comment:
Would it be feasible to test this particular addition in order to avoid regressions?

Reply:
@Szandor72 - I'm not sure how to test this from a functionality point of view. However, I did just notice that `RollupServiceTest.cls` had 4 failing test methods, with or without this change. They are all simple assertion failures. I am not sure if it is just my environment that they are failing in, so it would probably worth someone else taking a look as well. These are the test methods.

- `testLimitsConsumedWithConditions`
- `testLimitsConsumedWithSingleChildChanged`
- `testLimitsConsumedWithoutConditions`
- `testRollupWithoutChanges`

Reply:
The tests have passed on the feature build on this PR so I would assume the failures are related to your org shape. Thanks for bringing it to my attention.